cookbook/entities/acl.rst

```diff
@@ -0,0 +1,246 @@
+How to Protect Resources Using ACLs
```

Comment:
At this file you describe how to protect only the entities. So i think, the `How to Protect Entities Using ACLs ` is much better.